easyapplybot.py

```diff
@@ -77,24 +91,12 @@ def start_linkedin(self,username,password):
 		except TimeoutException:
 			print("TimeoutException! Username/password field or login button not found")
 
-	def wait_for_login(self):
-
-
-		time.sleep(1)
-
-		while True:
-			if self.browser.title != title:
-				print("\nStarting LinkedIn bot\n")
-				break
-			else:
-				time.sleep(1)
-				print("\nPlease Login to your LinkedIn account\n")
 
 	def fill_data(self):
 		self.browser.set_window_size(0, 0)
 		self.browser.set_window_position(2000, 2000)
```

Comment:
Instead of sending pixel window position, you might be able to use selenium to send keys for windows+direction
If it works, then it would dock the window to a side of the screen or another monitor.
I have not tried this, so I do not know if it will work. 

Ideally, I think we would want the window to not have a hard coded value due to not knowing everyone's personal display setup.